app/src/main/java/com/kickstarter/ui/activities/ProjectActivity.kt

```diff
@@ -553,14 +585,14 @@ class ProjectActivity : BaseActivity<ProjectViewModel.ViewModel>(), CancelPledge
 
     private fun setInitialRewardsContainerY() {
         val guideline = rewardsSheetGuideline()
-        rewards_container.y = (rewards_container.height - guideline).toFloat()
+        pledge_container_root.y = (pledge_container_root.height - guideline).toFloat()
         this.projectRecyclerView.setPadding(0, 0, 0, guideline)
     }
 
     private fun showBackingFragment(project: Project) {
         val (rewardsFragment, backingFragment) = supportFragmentManager.findFragmentById(R.id.fragment_rewards) as RewardsFragment to
                 supportFragmentManager.findFragmentById(R.id.fragment_backing) as BackingFragment
-        if(!rewardsFragment.isHidden && supportFragmentManager.backStackEntryCount == 0) {
+        if(!rewardsFragment.isHidden && supportFragmentManager.backStackEntryCount == 0 && !isFinishing) {
```

Comment:
What does `isFinishing` mean in this context? I might have missed where it was defined in this file.

Reply:
Oh it's a property of an `Activity` https://developer.android.com/reference/android/app/Activity 
<img width="882" alt="Screen Shot 2019-10-11 at 10 57 02 AM" src="https://user-images.githubusercontent.com/1289295/66661975-ee6b7980-ec15-11e9-8a96-40adc2d5bd5e.png">

So like when the user hits the back button, the `Activity` will call `finish`

Reply:
Ahh okay got it, thanks for the clarification!